rust/xaynet-client/src/participant.rs

```diff
@@ -5,7 +5,7 @@
 //! [client module]: ../index.html
 
 use std::default::Default;
-
+// TODO may remove MaskMany later
```

Comment:
What does this comment refer to?

Reply:
a stale comment, now removed. Thanks for spotting!